packages/databyss-ui/components/SearchContent/SearchContent.js

```diff
@@ -60,8 +60,9 @@ const SearchContent = () => {
               <View p="small" ml="small">
                 <Text>
                   <Highlighter
-                    searchWords={[query]}
-                    autoEscape
+                    searchWords={query
```

Comment:
Are we still using `Highlighter`? I thought we replaced this with the slate decoration API?

Reply:
as discussed, highlighter is a library plugin, unless we want to mount different instances of slatejs for each result